src/lib/helpers.js

```diff
@@ -0,0 +1,21 @@
+export const mustShowSelectionBar = state => state.ui.showSelectionBar || state.ui.selected.length !== 0
```

Comment:
My opinion is : if it compute something from state, it's a selector. Am I right @goldoraf ?